packages/extensions/venia-virtual-products/src/wrapUseCategory.js

```diff
@@ -0,0 +1,14 @@
+import { GET_CATEGORY } from './category.gql';
+
+const wrapUseCategory = original => props => {
+    const newProps = {
+        ...props,
+        operations: {
+            getCategoryQuery: GET_CATEGORY
```

Comment:
We no longer pass operations from the component and we [shallow merge](https://github.com/magento/pwa-studio/blob/develop/packages/peregrine/lib/talons/RootComponents/Category/useCategory.js#L44) this prop, so this may not be necessary. If this talon is used by external developers who have passed their own operations, maybe more relevant, but I don't actually anticipate anyone to install this extension besides ourselves.